.github/CODEOWNERS

```diff
@@ -165,7 +165,7 @@ content/en/code_analysis/                         @Datadog/static-analysis @Data
 content/en/quality_gates/                         @Datadog/ci-app-backend @Datadog/documentation
 
 # DDSQL Editor References
-content/en/ddsql_reference/*.md                     @Datadog/xpq @Datadog/documentation @Datadog/advanced-query-guild
+content/en/ddsql_reference/*.md                   @Datadog/documentation @Datadog/advanced-query-guild
```

Comment:
`xpq` reps are part of the `advanced-query-guild` group, so it's redundant to include them here twice